### PR TITLE
Folijet greening mg

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 1]
```diff
@@ -26,8 +27,50 @@ const verifyMessageOfDeteted = (quantity) => {
   InteractorsTools.closeCalloutMessage();
 };
 
+const columnName = {
+  status: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-status' })),
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:31]: Move all the same to const

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 2]
```diff
@@ -73,16 +116,17 @@ export default {
...
-      cy.do(Accordion('Errors in import')
-        .find(Checkbox({ id: 'clickable-filter-statusAny-committed' })).click());
-    }
+  selectYesfilterJobsByErrors: () => {
+    cy.do(Accordion('Errors in import')
+      .find(Checkbox({ id: 'clickable-filter-statusAny-error' })).click());
+  },
+
+  selectNofilterJobsByErrors: () => {
+    cy.do(Accordion('Errors in import')
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:128]: all the same to const

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 3]
```diff
@@ -110,14 +154,16 @@ export default {
...
-    if (filter === 'Yes') {
-      cy.do(singleRecordImportsAccordion
-        .find(Checkbox({ id: 'clickable-filter-singleRecordImports-yes' })).click());
-    } else {
-      cy.do(singleRecordImportsAccordion
-        .find(Checkbox({ id: 'clickable-filter-singleRecordImports-no' })).click());
-    }
+  selectNoFilterJobsByInventorySingleRecordImports: () => {
+    cy.do(singleRecordImportsAccordion
+      .find(Checkbox({ id: 'clickable-filter-singleRecordImports-no' })).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:159]: Move to const

File: cypress/support/fragments/data_import/matchOnVRN.js [hunk 4]
```diff
@@ -300,8 +300,8 @@ function createMatchProfileForVRN({
   cy.do(criterionSelection.select('Acquisitions data: Vendor reference number'));
 
   saveProfile();
-  cy.intercept('POST', '/data-import-profiles/matchProfiles').as('createMatchProfile');
-  cy.wait('@createMatchProfile');
+  //cy.intercept('POST', '/data-import-profiles/matchProfiles').as('createMatchProfile');
+  //cy.wait('@createMatchProfile');
```
Comment (penfriend) [inline cypress/support/fragments/data_import/matchOnVRN.js:304]: Do you need these comments?

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:31]: Done

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:128]: Done

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:159]: Done

File: cypress/support/fragments/data_import/matchOnVRN.js (hunk 4 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/matchOnVRN.js:304]: I deleted them

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 5]
```diff
@@ -28,9 +30,9 @@ const verifyMessageOfDeteted = (quantity) => {
 };
 
 const columnName = {
-  status: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-status' })),
-  jobProfile: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-jobprofilename' })),
-  runBy: MultiColumnList({ id:'list-data-import' }).find(MultiColumnListHeader({ id:'list-column-runby' }))
+  status: dataImportList.find(MultiColumnListHeader({ id:'list-column-status' })),
+  jobProfile: dataImportList.find(MultiColumnListHeader({ id:'list-column-jobprofilename' })),
+  runBy: dataImportList.find(MultiColumnListHeader({ id:'list-column-runby' }))
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:35]: MCLH also to const

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 6]
```diff
@@ -120,13 +122,11 @@ export default {
   },
 
   selectYesfilterJobsByErrors: () => {
-    cy.do(Accordion('Errors in import')
-      .find(Checkbox({ id: 'clickable-filter-statusAny-error' })).click());
+    cy.do(errorsInImportAccordion.find(Checkbox({ id: 'clickable-filter-statusAny-error' })).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:125]: Checkbox also to const

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 5 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:35]: All MCLH are different. What MCLH do you mean?

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 6 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:125]: All Checkboxes are different. What Checkbox do you mean?

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 7]
```diff
@@ -188,68 +215,45 @@ export default {
...
+          cy.expect(MultiColumnListCell({ content: or('Inventory Single Record - Default Create Instance', 'Inventory Single Record - Default Update Instance'), row: i }).exists());
+        }
+      });
+    } else {
+      return cy.get('#list-data-import').then(element => {
+        // only 100 records shows on every page
+        const resultCount = element.attr('data-total-count') > 99 ? 99 : element.attr('data-total-count');
+        // verify every string in result table
+        for (let i = 0; i < resultCount; i++) {
+          cy.expect(MultiColumnListCell({ content: or('Inventory Single Record - Default Create Instance', 'Inventory Single Record - Default Update Instance'), row: i }).absent());
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/logs/logsViewAll.js:247]: Done

**Review by MaksimTerekhov — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

**Pull request merged**